CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 ## Unreleased
 
+## v0.13.1
+
 IMPROVEMENTS:
 
 * Enable multiplexing [[GH-96](https://github.com/hashicorp/vault-plugin-auth-azure/pull/96)]
+* Upgrade to Go 1.20.2
+* Updated dependencies:
+   * `github.com/Azure/azure-sdk-for-go` v67.2.0+incompatible -> v68.0.0+incompatible
+   * `github.com/Azure/azure-sdk-for-go/sdk/azcore` v1.3.1 -> v1.4.0
+   * `github.com/Azure/azure-sdk-for-go/sdk/azidentity` v1.2.1 -> v1.2.2
```

Comment:
These dep updates going into a patch release worry me a bit while this plugin is bundled with Vault. Since v0.13.0 of this plugin has already been bundled into Vault 1.13, I wonder if these dep updates warrant a minor version increment of the plugin instead? (i.e. v0.14.0 instead of v0.13.1)

Reply:
No problem. I'm happy to do a minor release.